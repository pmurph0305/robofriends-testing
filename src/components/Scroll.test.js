import { shallow } from 'enzyme';
import React from 'react';
import Scroll from './Scroll';



describe('scroll testing', () => {
    let wrapper;
    beforeEach(() => {
        const mockProps = {
            children: {}
        }
        wrapper = shallow(<Scroll {...mockProps}/>)
    }) 
    test('expect to render Scroll Component', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('has correct props', () => {
        expect(wrapper.props().children).toEqual({})
        const mockProps2 = {
            children: {a:1}
        }
        const wrapper2 = shallow(<Scroll {...mockProps2}/>)
        expect(wrapper2.props().children).toEqual({a:1});
    })
})

