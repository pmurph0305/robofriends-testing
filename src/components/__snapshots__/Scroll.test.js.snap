// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`scroll testing expect to render Scroll Component 1`] = `ShallowWrapper {}`;
